src/resources/views/crud/fields/relationship/fetch_or_create.blade.php

```diff
@@ -12,7 +12,7 @@
     // and format it to JSON, so that select2 can parse it
     $current_value = old(square_brackets_to_dots($field['name'])) ?? old($field['name']) ?? $field['value'] ?? $field['default'] ?? '';
 
-    if ($current_value !== false) {
+    if ($current_value !== false && !empty($current_value)) {
```

Comment:
```suggestion
    if (!empty($current_value)) {
```

@pxpm `!empty` will include `!== false`, can we simplify this? 😅

Reply:
Hello @promatik !

From my tests `empty` would include `== false` (loose comparison) and not strict `===`.

So:
```php
empty(0) => true
```

While testing I found out that the current solution I provided did not cover the whole values (would left 0 (zero) out of equation because `!empty(0)` returns false. 

I've just refactored because we allow any non-empty values OR integers (as keys).

Let me know if this last change covers everything.

Best,
Pedro

Reply:
@pxpm  all good with `fetch.blade.php`, but with `fetch_or_create.blade.php` the rules should be the same right?